x-pack/plugins/observability_solution/slo/public/application.tsx

```diff
@@ -78,6 +78,13 @@ export const renderApp = ({
 
   const PresentationContextProvider = plugins.presentationUtil?.ContextProvider ?? React.Fragment;
 
+  const unregisterPrompts = plugins.observabilityAIAssistant.service.setScreenContext({
+    starterPrompts: [
+      { title: 'Getting started', icon: 'bullseye', prompt: 'What is an SLO?' },
+      { title: 'Getting started', icon: 'questionInCircle', prompt: 'Can you create an SLO?' },
```

Comment:
These still presumably need to be translated then. Also: the Assistant cannot create an SLO right? Generally, maybe good to add some responses to these prompts so we can figure out whether they are the right prompts?

Reply:
Yes, oversight